kinode.wit

```diff
@@ -85,10 +87,10 @@ interface standard {
 
     // Network errors come from trying to send a message to another node.
```

Comment:
Is it true that these can only happen when sending to another node? E.g. consider the case where I send a message to a local process and it times out. Does that not give back a SendError::Timeout?

If so, I recommend we change or remove this introductory sentence: I would assume, reading this first sentence, that it applied to this type in general (i.e. that a SendError is the same thing as a network error).